packages/app/config/babel.prod.js

```diff
@@ -4,33 +4,32 @@ module.exports = {
   // Don't try to find .babelrc because we want to force this configuration.
   babelrc: false,
   presets: [
+    require.resolve('@babel/preset-flow'),
     // Latest stable ECMAScript features
     [
       'env',
```

Comment:
It seems like this 'env' has been unchanged, but that should work too, right?

Reply:
Ah, let me fix that, just to be certain it uses the new stuff